### ISSUE TITLE
99 bottles program final folder doesn't seem to make sense

### ISSUE THREAD

**Issue by SinaKhalili** [rottytooth/Folders#4]
Hey there, cool esolang! I'm actually trying to write a python interpreter for it for fun 😄 

Anyways, and this may be something I'm not understanding, but in your example 99 bottles program the final command folder is `New Folder (4)` which has the following structure (limited to depth 3):

```
New folder (4)
     └── New folder (2)
                     └── New Folder
                     └── New Folder (2)
                     └── New Folder (3)
```

Doesn't every command folder need at least two subfolders? 
Also, assuming `New folder (2)` is an expression folder (because if we interpret it as such we get a ` LITERAL :: STRING :: 2 bottles of beer on the wall...`) then I'm assuming you'd want `New Folder (4)` to have a PRINT statement? 

For example I would have assumed the following structure:
```
New folder (4)
     └── New folder (1)
                     └── [ 4 subfolders to make it a PRINT statement ]
     └── New folder (2)
                     └── New Folder
                     └── New Folder (2)
                     └── New Folder (3)
```

Asking so I can make my interpreter spec-compliant lol

**Comment by rottytooth** [comment]
That is wonderful you are writing a Python interpreter for Folders; let me know how it goes!

You are exactly right, there was a missing New folder with four sub-directories. Or rather, my script that added .gitignores to the end nodes missed those directories, so git never picked them up. It is corrected now.

BTW, not sure how much you're actually looking at the C# code, but there's also an in-progress JS version (using brackets in the place of directories, for an online Folders emulator) that is quite a bit easier to follow here: https://github.com/rottytooth/Folders.JS

**Comment by SinaKhalili** [comment]
Hey @rottytooth I finally finished my Folders implementation in Python, [if you want to check it out](https://github.com/SinaKhalili/Folders.py) 😄  It was a fun little 2-day obsession for me lol. Although I had to change the 99bottles program a few more times, I hope it's still a correct implementation. At least, it captures the "spirit" lol. I honestly feel like I can "read" Folders now 😛 . I also transpiled (but to python). I didn't use all the 1337 functional JS tricks though - maybe in the future!

Cheers! And thanks for bringing this language to the people!
